docs/userguides/console.md

```diff
@@ -19,10 +19,7 @@ An example file might look something like this:
 from eth_utils import encode_hex, decode_hex
 
 
-def ape_init_extras(networks):
-    return {
-        "latest": lambda key: getattr(networks.active_provider.get_block("latest"), key)
-    }
+latest = lambda key: getattr(networks.active_provider.get_block("latest"), key)
```

Comment:
This actually causes issues with the linters, if they were to run.
anonymous methods are not meant to have name, so when you do `latest = <anon method>`, you might as well just do 

```python
def latest(key):
   return getattr(networks.active_provider.get_block("latest"), key)
```

Reply:
Far be it from me to argue with the linters.  Do you mind making that change?  I won't be able to until this evening.